utils/api/apiCalls.js

```diff
@@ -0,0 +1,5 @@
+export const getData = dataset => {
+  return fetch(`http://localhost:3001/api/v1/${dataset}`)
+    .then(response => response.ok ? response.json() 
+      : console.log(`Uh oh, something went wrong loading ${dataset}! Error: ${response.status}, ${response.statusText}`));
```

Comment:
Good job. David has reminded us how silly it is to just tell the user the error code, but I am still doing it that way. Thanks for the reminder.